policy/aggregation_type_config.go

```diff
@@ -63,14 +63,14 @@ type AggregationTypesConfiguration struct {
 	// Metric suffix for aggregation type median.
 	MedianSuffix *string `yaml:"medianSuffix"`
 
-	// Counter suffix override.
-	CounterSuffixOverride map[AggregationType]string `yaml:"counterSuffixOverride"`
+	// Counter suffix overrides.
+	CounterSuffixOverrides map[AggregationType]string `yaml:"counterSuffixOverride"`
```

Comment:
The yaml tag should also be `counterSuffixOverrides`, `timerSuffixOverrides`, and `gaugeSuffixOverrides`.

Reply:
yeah, good catch